pycbc/inference/likelihood.py

```diff
@@ -1075,4 +1075,5 @@ def loglr(self, **params):
                             MarginalizedPhaseGaussianLikelihood}
 
 __all__ = ['BaseLikelihoodEvaluator', 'TestNormal', 'TestEggbox', 'TestVolcano',
```

Comment:
Why is `__all__` being used here?

Reply:
So the things that are imported at the top of the file don't end up in the `inference` name space. It's a consequence of the `from foo.bar import *` that we do in modules' `__init__.py` files. I agree it's not a great setup, but seems to be the standard in pycbc.